CI/travis.before_install.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
-if [ -z "${TRAVIS_OS_NAME}" ] || [ "${TRAVIS_OS_NAME}" = "linux" ]; then
-  echo Before install on linux.
```

Comment:
I may be mistaken - but it looks to me as the "system" reports the GCC and other item versions at the start of the run but that may not necessarily be correct if we have forced that or some other component to be updated during the _before install_ stage...